src/test/java/org/ibp/api/java/impl/middleware/dataset/DatasetServiceImplTest.java

```diff
@@ -1084,4 +1084,15 @@ public void testGetReplaceObservationUnitsEntry_Ok() {
 		Mockito.verify(this.middlewareDatasetService, times(1)).replaceObservationUnitEntry(eq(itemIds), eq(newEntryId));
 	}
 
+	@Test
+	public void testAddLocationVariables() {
+		final List<MeasurementVariable> measurementVariableList = new ArrayList<>();
+		this.studyDatasetService.addLocationVariables(measurementVariableList);
+		Assert.assertEquals(2, measurementVariableList.size());
```

Comment:
Create test case for scenario that location abbreviation is already present in parameter of variables